source/dev-setup.rst

```diff
@@ -59,6 +59,12 @@ extensions::
     brew install postgres
     brew install postgis
 
+Note that "brew install postgis" will install the latest version of postgis along with the latest
+version of potstgres, so if you have a previous installation of postgres you may end up with two
+versions installed causing conflicts and connection issues. It is recommended, if you have a
+previous installation, that you uninstall this version version. Migration of existing data is
+left as an exercise for the reader.
```

Comment:
We could suggest the first answer at http://stackoverflow.com/questions/24379373/how-to-upgrade-postgres-from-9-4-to-9-5-without-losing-data (skipping the final step that upgrades the ruby gem :-) )